Functions/Deploy-GomRepo.ps1

```diff
@@ -33,6 +33,18 @@ function Deploy-GomRepo {
         New-GitHubRepository -OrganizationName $OrganizationName -RepositoryName $RepoName | Format-List
     }
 
+    if($RepoConfig.CodeOwners){
+        $CodeOwnersContent = ""
+        foreach ($property in $($RepoConfig.CodeOwners | Get-Member -MemberType Properties)){
+            $path = $($property.Name)
+            $owners = $RepoConfig.CodeOwners.$path
+            $CodeOwnersContent += "$path $owners`n"
+        }
+        Write-Output "Complete CODEOWNERS content is`n$CodeOwnersContent"
+        Set-GitHubContent -Path .github/CODEOWNERS -RepositoryName $RepoName -OwnerName $OrganizationName -Content $CodeOwnersContent -CommitMessage "Added CODEOWNERS"
+        Write-Output "Wrote CODEOWNERS file for $RepoName to .github/CODEOWNERS"
```

Comment:
```suggestion
        Write-Host "Wrote CODEOWNERS file for '$RepoName' to .github/CODEOWNERS"
```

This message is not something we care to pass down the pipeline https://stackoverflow.com/questions/19754069/powershell-difference-between-write-host-and-write-output